lib/msf/ui/console/driver.rb

```diff
@@ -601,6 +605,15 @@ def handle_ssh_ident(val)
     $VERBOSE = verbose
   end
 
+  def handle_session_tlv_logging(val)
+    val = val.downcase
+    if val && (val == 'console' || val == 'false' || val =~ /^file:.+/)
```

Comment:
```suggestion
    if val && (val.casecmp?('console') || val == 'false' || val.start_with?('file:'))
```

https://apidock.com/ruby/v2_5_5/String/casecmp%3F